micro-rdk/src/common/robot.rs

```diff
@@ -254,6 +265,11 @@ impl LocalRobot {
         let mut robot = LocalRobot {
             executor: exec,
             part_id,
+            cloud_metadata: config.cloud.as_ref().map(|cfg| CloudMetadata {
+                org_id: cfg.primary_org_id.clone(),
+                location_id: cfg.location_id.clone(),
+                machine_id: cfg.machine_id.clone(),
```

Comment:
is part_id == machine_id?

Reply:
no, there's a separate "machine_id" and "machine_part_id"